### PR TITLE
feat(GiniHealthSDK): Pay invoice to new payment review screen IPC-133

### PR DESCRIPTION
**Pull request by razvancapra**
- Pay invoice button from payment component open the new Payment review screen: 
- Customize the look of the Payment Review Screen with Gini Health Configuration
- Remove check of payment providers installed + add new flow on Start with Gini Capture & Start with test document
- English version of SDK

IPC-133
IPC-153
IPC-81
IPC-167
IPC-206

https://ginis.atlassian.net/browse/IPC-133
https://ginis.atlassian.net/browse/IPC-153
https://ginis.atlassian.net/browse/IPC-81
https://ginis.atlassian.net/browse/IPC-167
https://ginis.atlassian.net/browse/IPC-206

### CODE REVIEW

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentReviewViewController.swift [hunk 1]
```diff
@@ -208,18 +179,19 @@ public final class PaymentReviewViewController: UIViewController, UIGestureRecog
...
     }
     
     // MARK: - Info bar
 
     fileprivate func configureInfoBar() {
-        infoBar.roundCorners(corners: [.topLeft, .topRight], radius: giniHealthConfiguration.infoBarCornerRadius)
-        infoBar.backgroundColor = UIColor.from(giniColor: giniHealthConfiguration.infoBarBackgroundColor)
-        infoBarLabel.textColor = UIColor.from(giniColor: giniHealthConfiguration.infoBarTextColor)
-        infoBarLabel.font = giniHealthConfiguration.customFont.regular
+        infoBar.roundCorners(corners: [.topLeft, .topRight], radius: 12.0)
```
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentReviewViewController.swift:191]: [Suggestion]: could you please move radius: 12.0 to the Constants?

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentReviewViewController.swift [hunk 2]
```diff
@@ -233,7 +205,7 @@ public final class PaymentReviewViewController: UIViewController, UIGestureRecog
                        delay: 0, usingSpringWithDamping: 1.0,
                        initialSpringVelocity: 1.0,
                        options: [], animations: {
-                           self.infoBar.frame = CGRect(x: 0, y: self.inputContainer.frame.minY + self.giniHealthConfiguration.infoBarCornerRadius - self.infoBar.frame.height, width: screenSize.width, height: self.infoBar.frame.height)
+                           self.infoBar.frame = CGRect(x: 0, y: self.inputContainer.frame.minY + 32 - self.infoBar.frame.height, width: screenSize.width, height: self.infoBar.frame.height)
```
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentReviewViewController.swift:208]: [Suggestion]: could you please move 32 to the Constants ?

File: HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoicesListViewModel.swift [hunk 3]
```diff
@@ -158,25 +159,26 @@ final class InvoicesListViewModel {
...
         self.coordinator.invoicesListViewController.navigationController?.pushViewController(paymentInfoViewController, animated: true)
     }
     
-    func didTapOnBankPicker(documentID: String?) {
-        guard let documentID else { return }
-        Log("Tapped on Bank Picker on :\(documentID)", event: .success)
+    func didTapOnBankPicker(documentId: String?) {
+        guard let documentId else { return }
+        Log("Tapped on Bank Picker on :\(documentId)", event: .success)
         let paymentProvidersBottomViewController = paymentComponentsController.bankSelectionBottomSheet()
```
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoicesListViewModel.swift:173]:
```suggestion
        let bankSelectionBottomSheet = paymentComponentsController.bankSelectionBottomSheet()
```

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/UITextField+Utils.swift
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/UITextField+Utils.swift:1]: Can we move the whole file to the extension folder?

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentsController.swift
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentsController.swift:1]: [Change request] we need to move `GiniDocument` protocol to the Example app, customer can have any document structure, the must to have documentId: String, that we pass further 🙏

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Great job 👏🏻 @razvancapra 
Please find a few small comments:
Could you please cleanup our Payment Review Screen, I didn't refactor for a long time and move out the constants in the enum?

**Comment by razvancapra** [comment]
@zladzeyka addded the info bar fix here as well because is a minor fix: 
https://ginis.atlassian.net/browse/IPC-81

File: HealthSDK/GiniHealthSDK/Documentation/source/Integration.md [hunk 4]
```diff
@@ -76,10 +76,8 @@ let partialDocs = [PartialDocumentInfo(document: createdDocument.links.document)
 
 ##  Check preconditions
 
-There are three methods in GiniHealth:
+There is one method in GiniHealth:
```
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDK/Documentation/source/Integration.md:79]:
```suggestion
There is a method in GiniHealth:
```

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/en.lproj/Localizable.strings [hunk 5]
```diff
@@ -7,17 +7,18 @@
...
 "ginihealth.reviewscreen.next.button.title" = "Next";
+"ginihealth.reviewscreen.banking.app.button.label" = "To the banking app";
 "ginihealth.errors.no.banking.app.installed" = "Your banking app does not support this service. Have you tried ING app yet?";
-"ginihealth.reviewscreen.infobar.message" = "Please review the extracted information below";
+"ginihealth.reviewscreen.infobar.message" = "Please check the pre-filled data.";
 "ginihealth.errors.default" = "Oops something went wrong. Please try again.";
 "ginihealth.errors.failed.payment.request.creation" = "Oops something went wrong. Please try again.";
 "ginihealth.errors.failed.recipient.non.empty.check" = "Recipient is required.";
 "ginihealth.errors.failed.iban.non.empty.check" = "IBAN is required.";
 "ginihealth.errors.failed.iban.validation.check" = "IBAN is not valid";
```
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/en.lproj/Localizable.strings:20]:
```suggestion
"ginihealth.errors.failed.iban.validation.check" = "IBAN is not valid.";
```

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/en.lproj/Localizable.strings [hunk 6]
```diff
@@ -7,17 +7,18 @@
...
-"ginihealth.errors.failed.amount.non.empty.check" = "Amount can't be 0";
+"ginihealth.errors.failed.amount.non.empty.check" = "Invalid.";
 "ginihealth.errors.failed.purpose.non.empty.check" = "Purpose is required.";
 "ginihealth.errors.failed.default.textfield.validation.check" = "The field is not valid";
```
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/en.lproj/Localizable.strings:23]:
```suggestion
"ginihealth.errors.failed.default.textfield.validation.check" = "The field is not valid.";
```

Comment (zladzeyka) [reply HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/en.lproj/Localizable.strings:23]: do we use "ginihealth.errors.failed.default.textfield.validation.check" = "The field is not valid.?

**Review by zladzeyka — COMMENTED** [review]
I've left 2 tiny comments 😇

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/en.lproj/Localizable.strings (hunk 6 above)
Comment (razvancapra) [reply HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/en.lproj/Localizable.strings:23]: no, I'll remove it and clean other 2 langs that are not used

**Review by zladzeyka — APPROVED** [review]
Looks good!

**Review by zladzeyka — APPROVED** [review]

**Pull request merged**